website/documentation/appendix/releasenotes/0.10.0-M1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Release Notes
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.  
+Here are some of the major highlights:
+
+- **Web Editor** that allows to create information, function blocks and entity type models through the web browser
+- New **Code Generators** have been added to the list of existing generators.
+- Bug fixes and improvements
+
+
+### Web Editor
+With the **Web Editor**, you can create individual projects where you can describe the characteristics and functionality of the Information Models and publish then to Vorto Repository for others who could use to generate code using Code Generators. This is solely done using a web browser.
```

Comment:
With the web editor, device manufacturers can describe devices online and publish these as information models to the Vorto Repository.